### PR TITLE
Add CF deployment artifact details

### PR DESCRIPTION
**Pull request by gregturn**

### CODE REVIEW

**Comment by gregturn** [comment]
Related to https://github.com/spinnaker/orca/pull/668 and https://github.com/spinnaker/clouddriver/pull/209

File: app/scripts/modules/cloudfoundry/serverGroup/configure/serverGroupArtifactSettingsDirective.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      <b ng-if="!artifactSettingsCtrl.isS3()">Password</b>
+      <b ng-if="artifactSettingsCtrl.isS3()">Secret Key</b>
+    </div>
+    <div class="col-md-7">
+      <input type="password"
+             class="form-control input-sm"
+             name="password"
+             ng-model="command.password"/>
+    </div>
+  </div>
```
Comment (anotherchrisberry) [inline app/scripts/modules/cloudfoundry/serverGroup/configure/serverGroupArtifactSettingsDirective.html]: You can remove a lot of the boilerplate by using the `stageConfigField` directive, e.g.
```html
<div class="form-group">
  <div class="col-md-3 sm-label-left">
    <b>Artifact</b>
  </div>
  <div class="col-md-7">
    <input type="text"
           class="form-control input-sm"
           name="artifact"
           ng-model="command.artifact"/>
  </div>
</div>
```
becomes
```html
<stage-config-field label="Artifact">
  <input type="text"
         class="form-control input-sm"
         name="artifact"
         ng-model="command.artifact"/>
</stage-config-field>
```

**Comment by anotherchrisberry** [comment]
Aside from the `<stage-config-field>` changes, it looks good to me!

File: app/scripts/modules/cloudfoundry/serverGroup/configure/serverGroupArtifactSettingsDirective.html (hunk 1 above)
Comment (gregturn) [inline app/scripts/modules/cloudfoundry/serverGroup/configure/serverGroupArtifactSettingsDirective.html]: @anotherchrisberry Can I override the label's class value? I noticed that "Access Key" and "Secret Key" were a bit too long for col-md-2, so mine used col-md-3.

Comment (anotherchrisberry) [inline app/scripts/modules/cloudfoundry/serverGroup/configure/serverGroupArtifactSettingsDirective.html]: Whoops, I'm an idiot - this isn't a stage, it's the actual modal. Sorry, disregard the whole comment.

**Comment by anotherchrisberry** [comment]
LGTM - do you want me to merge it or hold off until the related PRs are ready?

**Comment by gregturn** [comment]
We probably should. This is the last step.

**Comment by gregturn** [comment]
@anotherchrisberry This pulls in load balancer support along with configuring CF server groups with an extra "Load Balancer" step. I've verified it works against https://github.com/spinnaker/clouddriver/pull/192

**Comment by anotherchrisberry** [comment]
LGTM - let me know when you're ready for it to be merged (also, looks like it needs to be rebased)

**Pull request merged**